### PR TITLE
Editing Video with Python + MoviePy (tutorial) (non-conflict PR)

### LINKED ISSUE #1234: Editing Video with Python (tutorial)
### Brief Summary: 
Videos are great for getting ideas across. But editing those videos can take a long time. Some types of videos (from sports clips to data visualizations) are both simple and time-consuming to edit. In this tutorial, we will use MoviePy, a library for Python, to automatically edit and add effects to a given video clip.

### Key Takeaways: 
- Recognize situations where MoviePy would be useful in the workplace
- Understand the key concepts/objects in MoviePy
- Use MoviePy to build an automatic video-editing script

### References: 
- https://zulko.github.io/moviepy/
- https://www.learningcontainer.com/mp4-sample-video-files-download/
- https://medium.com/@TejasBob/moviepy-is-awesome-part1-f90e91fffbb9

### Prerequisites:
- Basic Python (3) knowledge

### PR DESCRIPTION
**Pull request by nicholaskross**
#### Preliminary Checks - Formatting and Structure 
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**
- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**
- [x] Have you included a hero image that is 600x400 pixels and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
- [x] In your PR - add a comment that states **["This fixes #(enter your Idea Content Form #)"](https://youtu.be/TKJ4RdhyB5Y?t=278)** to link and close your Idea Content (issue) when the PR is merged.

**NOTE: (1st time contributors - submit ONLY one article to get familiar with the process and style-once published, you can then submit any number of articles.)**
      
#### Grammar Checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) -  [Grammarly](https://grammarly.com) 
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical Checks
- [x] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**

### Contribution Guidelines
For first-time contributors and for more details, **see [Contributing Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**

Tutorial on editing videos in Python, using the MoviePy library.

This fixes #1234. It also replaces [the old PR](https://github.com/section-io/engineering-education/pull/1494).

### CODE REVIEW

**Comment by katungi** [comment]
@nicholaskross Has this article gone through review before?

**Comment by nicholaskross** [comment]
@katungi The article has not been reviewed yet.

**Review by katungi — COMMENTED** [review]
Interesting article @nicholaskross Preliminary has begun.

**Review by katungi — COMMENTED** [review]
Amazing article , can't wait to start the tech review and get my hands dirty. I have a problem with the questions in the article. Can we check on that. Since you answering the questions right next to the questions, I think the article would flow better without them.

I am now proceeding with the tech review

File: articles/video-editing-python-moviepy/index.md [hunk 1]
```diff
@@ -0,0 +1,240 @@
...
+
+### Prerequisites
+
+You should know the basics of Python 3 (including setting up projects). Basic knowledge about video files is helpful, but not necessary. This tutorial uses Python 3.7.4, MoviePy 1.0.3, PyCharm 2019.3.5 (Community Edition), and Windows 10.
+
+### Why MoviePy? And When?
+
+MoviePy is a Python module for editing videos. It can cut and arrange clips, add video effects, and edit audio. It can work like a toolbox, if you only make changes to one clip. Or, if you have to edit lots of clips, MoviePy can automate that process.
+
+Where would you use MoviePy in a workplace setting? It works best when every video is similar and simple to edit. For example, if you work for a TV studio, you might need to put together lots of clips from a football game. MoviePy could split a full game (hours of video) into a few clips. Then, it could overlay your company's watermark in the corner.
```
[the remark this answers is not in the document]
Comment (nicholaskross) [reply articles/video-editing-python-moviepy/index.md]: Fixed!

**Comment by katungi** [comment]
Thanks for making the changes. I am currently going through the tech review. Should be done soon

**Comment by katungi** [comment]
Hey @nicholaskross done with the tech review. It was a lot of information to consume all at once. But everything looks good to me.
@hectorkambow This article looks good. Ready for final review.

**Pull request merged**